### PR TITLE
functionality added to all pages

### PR DESCRIPTION
**Pull request by SumitSaundadkar**
After adding pages I have provided functionality to the pages.

https://bucolic-chebakia-ba4c9b.netlify.app/

### CODE REVIEW

**Review by TusharJade — APPROVED** [review]
Everyhting looks great just look at onclick on like its color is not coming back as on like other that good to go for merge 👍

File: src/pages/watchLater/Watchlater.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const WatchLaterPage=()=>{
+    const {videoListState}=useFilterContext();
+    const  watchLaterList=[...videoListState.watchLater].reverse();
+
+    return(
+        <div>
+             <NavBar/>
+             <SideBar/>
+           <div className='watch_container'>
+             <h1>Watche Later</h1>
```
Comment (Ananyamadhu08) [inline src/pages/watchLater/Watchlater.jsx]: There is a typo here in the watch later heading.

File: src/pages/watchLater/Watchlater.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { NavBar } from '../../components/Navbar/Navbar';
```
Comment (Ananyamadhu08) [inline src/pages/watchLater/Watchlater.jsx:1]: you can use barrel exports.

File: src/pages/watchLater/watchlater.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+     position: absolute;
+     flex-wrap: wrap;
+     top: 15%;
+     left: 18%;
+     color: var(--hover-bg);
+}
+.watch_video{
+    display: flex;
+    flex-direction: row;
+    gap: 16px;
```
Comment (Ananyamadhu08) [inline src/pages/watchLater/watchlater.css]: can use rem instead of px.

**Review by Ananyamadhu08 — APPROVED** [review]
I have suggested some changes please look into those, other than that your code looks good.  You can go ahead and merge!

File: src/pages/watchLater/watchlater.css (hunk 3 above)
Comment (SumitSaundadkar) [reply src/pages/watchLater/watchlater.css]: done

File: src/pages/watchLater/Watchlater.jsx (hunk 1 above)
Comment (SumitSaundadkar) [reply src/pages/watchLater/Watchlater.jsx]: done

**Pull request merged**